subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/android/RealLifeAndroidDexingTransformsPerformanceTest.groovy

```diff
@@ -34,17 +35,19 @@ class RealLifeAndroidDexingTransformsPerformanceTest extends AbstractCrossBuildP
     public static final String DEXING_TASK = "dexing task"
 
     @Unroll
-    def "dexing task vs transform: #tasks on #testProject without (android/transform) cache"() {
+    def "dexing task vs transform: #tasks on #testProject #withOrWithout (android/transform) cache"() {
         given:
-        def invocationOptions = [tasks: tasks, memory: memory, enableAndroidBuildCache: false]
+        def invocationOptions = [tasks: tasks, memory: memory, enableAndroidBuildCache: enableCaches]
 
         runner.testGroup = "Android dexing"
         runner.buildSpec {
             projectName(testProject)
             displayName(DEXING_TRANSFORM)
             warmUpCount warmUpRuns
             invocationCount runs
-            listener(cleanTransformsCache())
+            if (!enableCaches) {
+                listener(cleanTransformsCacheBeforeInvocation())
+            }
```

Comment:
I think it would make sense to add some comments about why we are doing it like this. I.e. that we are trying to emulate no cache by deleting the transform caches (and mention the overhead caused by storing in the transform cache).

Reply:
I added a comment why we are deleting the cache.

Reply:
Not copying anything yet, you're right, but we will.

Reply:
...but then we can simply switch off the build cache completely. OK, disregard my comment about overhead. :)